src/components/editMeasure/populationCriteria/QDMReporting/QDMReporting.tsx

```diff
@@ -220,6 +221,7 @@ const QDMReporting = () => {
                     aria-labelledby="improvement-notation-other-label"
                     type="text"
                     disabled={!canEdit}
+                    required={true}
```

Comment:
If this is a required field, then we do need to provide that information to user.

1. We need Asterisk (*) for this field.
2. We need to use display helperText by using formik.errors.improvementNotationOther field